examples/tree/plot_unveil_tree_structure.py

```diff
@@ -44,7 +44,9 @@
 #
 # The decision classifier has an attribute called ``tree_`` which allows access
 # to low level attributes such as ``node_count``, the total number of nodes,
-# and ``max_depth``, the maximal depth of the tree. It also stores the
+# and ``max_depth``, the maximal depth of the tree. To compute the depth of each
```

Comment:
The new sentence says the same thing twice. I suggest:

> The `tree_.compute_node_depths()` method computes the depth of each node in the tree.

Reply:
Changed to this.